docs/apis/subsystems/communication/index.md

```diff
@@ -149,6 +157,35 @@ This is destructive and might remove all the messages and other data associated
 
 :::
 
+### Create and configure a room according to the provider
+
+There are cases where the provider is changed for a communication instance. For example, previously the provider was set to _Provider A_ and now it has changed to _Provider B_. In this case, the room and its members need to be configured according to the new provider. Without any extra logic needed, the method `$communication->configure_room_and_membership_by_provider()` takes care of this in the following way:
+
+1. Communication provider is changed from _Provider A_ to _Provider B_.
+2. New room is created in _Provider A_.
```

Comment:
Is this correct? From the example I understood that Provider B was the new one, so I find it weird creating a new room in Provider A...

Reply:
A and B typo, resolved.